vendor/bootstrap-colorpicker.css

```diff
@@ -1,5 +1,14 @@
 /*!
- * Bootstrap Colorpicker v2.5.2
+ * Bootstrap Colorpicker v2.5.1
```

Comment:
A downgrade? 🤔

Reply:
Yes. They changed how they show the colorpicker UI and I couldn't be bothered trying to make it work.
So I reverted to the same version we use the Appearance tab.